docs/skywalking/skywalking.md

```diff
@@ -0,0 +1,52 @@
+# SkyWalking
+
+Skywalking-manager is a handler plugin of mesher, it reports tracing data to skywalking server
+
+## Configurations
+**In conf/mesher.conf**
+
+**servicecomb.apm.tracing.enable**
+>  *(optional, bool)* enable apm
+
+**servicecomb.apm.tracing.serverUri**
+>  *(optional, string)* server address of skywalking
+
+## Example
+```yaml
+servicecomb:
+  apm:
+    tracing:
+      enable: true
+      serverUri: 127.0.0.1:11800
+```
+
+## SkyWawlking-Manager Init
+**In file proxy/bootstrap/bootstrap.go**
+```shell script
+import "github.com/apache/servicecomb-mesher/proxy/pkg/skywalking"
+```
+**In function Start()**
```

Comment:
should teach how to use configuration file， skywalking
 is not enabled in chain by default

Reply:
and do not teach so many code that you already wrote

Reply:
Modify according to code review comments